lms/djangoapps/support/static/support/jsx/program_enrollments/inspector.jsx

```diff
@@ -2,55 +2,174 @@ import React from 'react';
     </div>
-    <Button label="Search" type="submit" className={['btn', 'btn-primary']} />
-  </form>
+    <hr />
+  </div>
+);
+
+const renderEnrollmentsSection = enrollments => (
+  <div>
+    <h3>
+      Program Enrollments
+    </h3>
+    {enrollments.map(enrollment => (
+      <div key={enrollment.program_uuid}>
+        <div>Program {enrollment.program_uuid} Enrollments</div>
+        <div> <span>Status: </span> {enrollment.status} </div>
+        <div> <span>Created: </span> {enrollment.created} </div>
+        <div> <span>Last updated: </span> {enrollment.modified} </div>
+        <div> <span>External User Key: </span> {enrollment.external_user_key} </div>
+        {enrollment.program_course_enrollments && enrollment.program_course_enrollments.map(
+          programCourseEnrollment => (
+            <div key={programCourseEnrollment.course_key}>
+              <br />
+              <h4>Course {programCourseEnrollment.course_key}</h4>
+              <div> <span>Status: </span> {programCourseEnrollment.status} </div>
+              <div> <span>Created: </span> {programCourseEnrollment.created} </div>
+              <div> <span>Last updated: </span> {programCourseEnrollment.modified} </div>
+              {programCourseEnrollment.course_enrollment && (
+                <div>
+                  <br />
+                  <h4>Linked course enrollment</h4>
+                  <div><span>Course ID: </span>
+                    {programCourseEnrollment.course_enrollment.course_id}
+                  </div>
+                  <div> <span>Is Active: </span>
+                    {programCourseEnrollment.course_enrollment.is_active}
+                  </div>
+                  <div> <span>Mode / Track: </span>
+                    {programCourseEnrollment.course_enrollment.mode}
+                  </div>
+                </div>
+              )}
+            </div>
+          ))}
+      </div>
+    ))}
+    <hr />
+  </div>
+);
+
+
+export const ProgramEnrollmentsInspectorPage = props => (
+  <div>
+    {JSON.stringify(props.learnerInfo) !== '{}' && (<h2> Search Results </h2>)}
+    {props.learnerInfo.user &&
+      renderUserSection(props.learnerInfo.user)}
+    {props.learnerInfo.id_verification &&
+      renderVerificationSection(props.learnerInfo.id_verification)}
+    {props.learnerInfo.enrollments &&
+      renderEnrollmentsSection(props.learnerInfo.enrollments)}
+    <form method="get">
+      <h2>Search For A Masters Learner Below</h2>
+      {props.errors.map(errorItem => (
+        <StatusAlert
+          open
+          dismissible={false}
+          alertType="danger"
+          dialog={errorItem}
+        />
+      ))}
+      <div key="edX_accounts">
+        <InputText
+          name="edx_user"
+          label="edX account username or email"
+        />
+      </div>
+      <div key="school_accounts">
+        <InputSelect
+          name="IdPSelect"
+          label="Learner Account Providers"
+          value="Select One"
```

Comment:
Can you confirm for me that the default value shows up for you?

On my devstack, I see the following.

![image](https://user-images.githubusercontent.com/11871801/77179444-e1916280-6a9e-11ea-94c2-5750afb35cd9.png)

Reply:
The reason you are seeing this is because you don't have any IdP configured on your environment. You would need to configure a SAML IDP on your local machine, and link it to an organization properly for any value to show up in the select dropdown.

Reply:
Shouldn't I see the default value "Select One", though? "Select One" makes me thing this is the value you want to appear in the component when nothing is selected, is that right, or am I misunderstanding?

Reply:
I think the value here is no longer needed. It doesn't show up because it got overridden by the `options` attribute. I will remove this values assignment. I don't think we need a `select one` option here.